src/main/java/uk/gov/hmcts/reform/em/stitching/domain/DocumentTask.java

```diff
@@ -35,6 +36,9 @@
     @OneToOne(cascade = CascadeType.ALL)
     private Callback callback;
 
+    @Column(name = "case_data")
+    private String caseData;
```

Comment:
This property needs to be of unlimited length. We can't assume any length for the case JSON. I'd also rename it to coverpageTemplateData and move it to Bundle class because:
- the Bundle class already has coverpageTemplate property. 
- we keep Stitching API client agnostic as much as possible
- caseData suggests a client, but at the same time is of type String which is very vague